Customization/T200/PhoneRepairShop_Code/PhoneRepairShop_Code/PhoneRepairShop_Code.csproj

```diff
@@ -49,22 +49,18 @@
     <Reference Include="PX.Objects">
       <HintPath>..\..\..\..\Bin\PX.Objects.DLL</HintPath>
     </Reference>
-    <Reference Include="PX.Web.Customization, Version=1.0.0.0, Culture=neutral, PublicKeyToken=3b136cac2f602b8e, processorArchitecture=MSIL">
```

Comment:
Please restore these lines. They are necessary for execution of the customization plug in from the Data folder.
Please also rebuild the project with the Data folder and this reference included. Then update the bin file in the customization project.